channels/in.m3u

```diff
@@ -194,7 +216,7 @@ http://103.199.161.254/Content/jaihind/Live/Channel(Jaihind)/index.m3u8
 #EXTINF:-1 tvg-id="JanTV.in" tvg-name="Jan TV" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Jan TV (360p)
 http://jantvstream.in:1935/edge1/sc1jantv.stream_aac/playlist.m3u8
 #EXTINF:-1 tvg-id="JanamTV.in" tvg-name="Janam TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/LWUpmDm.png" group-title="News",Janam TV (720p)
-https://vidcdn.vidgyor.com/janamtv-origin/liveabr/playlist.m3u8
```

Comment:
We don't remove alternate sources if those work well

Reply:
Those links were taking time 
, That's why updated the new links. Can you fix it now?

Reply:
You can add them back into your file, just add them as another channel with the same description

Reply:
Added those links as new channels.

Reply:
Looks like you have forgot to replace links to the old ones where it's needed

Reply:
Oh damn. how do I fix it now?

Reply:
Fixed it.But PR test failed again.

Reply:
there you go. You can check linter logs to see where there trailing space or the wrong tag is